src/stack.ts

```diff
@@ -580,8 +580,8 @@ class StackConverter extends ArtifactConverter {
     private resolveIntrinsic(fn: string, params: any) {
         switch (fn) {
             case 'Fn::GetAtt': {
-                debug(`Fn::GetAtt(${params[0]}, ${firstToLower(params[1])})`);
-                return this.resolveAtt(params[0], firstToLower(params[1]));
+                debug(`Fn::GetAtt(${params[0]}, ${params[1]})`);
+                return this.resolveAtt(params[0], params[1]);
```

Comment:
Inside the `resolveAtt` function we use the `toSdkName` function which already handles name conversion. Using `firstToLower` here messed it up.